kani-compiler/src/main.rs

```diff
@@ -65,14 +60,41 @@ fn rustc_gotoc_flags(lib_path: &str) -> Vec<String> {
         "crate-attr=feature(register_tool)",
         "-Z",
         "crate-attr=register_tool(kanitool)",
+    ];
+    args.iter().map(|s| s.to_string()).collect()
+}
+
+/// This function adds Kani library args.
+fn kani_library_args(lib_path: &str) -> Vec<String> {
+    // The option below provides a mechanism by which definitions in the
+    // standard library can be overriden. See
+    // https://rust-lang.zulipchat.com/#narrow/stream/182449-t-compiler.2Fhelp/topic/.E2.9C.94.20Globally.20override.20an.20std.20macro/near/268873354
+    // for more details.
+    let mut base_args = base_kani_args();
+    let lib_path_kani: PathBuf = [lib_path, "kani"].iter().collect();
```

Comment:
nit: this code for the kani library is a bit redundant. Not a must have, but we might want to eventually reuse this code.